### DISCUSSION TITLE
New Raspberry Pi 4 install - no LEDs

### DISCUSSION THREAD

**Discussion by SpencerDStone** [awawa-dev/HyperHDR#722] [General]
I have everything up and running. I have the data cable plugged into GPIO 18 (physical 6.) I can see my video being pulled, but the LEDs do nothing. They are new and hooked up to a sepreate power supply. I have 5v at the power wires, but I read 4v on the signal wire.

**Comment by awawa-dev** [comment]
You are using a solution that is not supported here, but in the wiki->FAQ you may find some hints on how to configure it and what to pay attention to in the HyperHDR logs. You won't be able to measure what's happening on the data line using a multimeter, it almost certainly won't show you even peak voltages, you need an oscilloscope. Also make sure you have common ground.

**Reply by SpencerDStone** [reply]
This was the solution - https://github.com/awawa-dev/HyperHDR/discussions/43#discussioncomment-441244

I'll test it on my other test rig, but everything is working on my school test.

**Reply by awawa-dev** [reply]
Yes, this is also mentioned in the FAQ. But it is based on the pi user, it was long ago removed from the official Raspberry Pi images, so you can't use copy-paste and voilà: you have to read what is happening in the logs and what user the service is running on... But as I already wrote, this is not a solution officially supported here.
